recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: mpld3-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/m/mpld3/mpld3-{{ version }}.tar.gz
-  sha256: b080f3535238a71024c0158280ab4f6091717c45347c41c907012f8dd6da1bd5
+  sha256: c589db8b661aee25c93e198e2e18ed47b9a96951de41d96241345acec5f819ab
 
 build:
   number: 0
```

Comment:
> The package was left as an architecture specific package due to the version change going from 0.5.5 to 0.5.7 this is a build number change.

This is not a build number change in the conda sense; rather, it's an upstream patch/revision level change (assuming upstream follows SemVer). conda-forge's recipe has been [noarch since 2018](https://github.com/conda-forge/mpld3-feedstock/commit/d94be77aa), so I don't understand why we're waiting to convert Anaconda's over to noarch.

Reply:
> > The package was left as an architecture specific package due to the version change going from 0.5.5 to 0.5.7 this is a build number change.
> 
> This is not a build number change in the conda sense; rather, it's an upstream patch/revision level change (assuming upstream follows SemVer). conda-forge's recipe has been [noarch since 2018](https://github.com/conda-forge/mpld3-feedstock/commit/d94be77aa), so I don't understand why we're waiting to convert Anaconda's over to noarch.

That is a valid point, per your request I have modified the package to work as a noarch